keyboards/spaceman/pancake/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,71 @@
+#include QMK_KEYBOARD_H
+enum layers {
+  _DEFAULT,
+  _LOWER,
```

Comment:
I don't see a `_LOWER` layer here.

Reply:
This is a artifact from an old version the keymap removing now.